src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -117,6 +144,45 @@ object VersionInjectorPlugin extends AutoPlugin {
         gitConfFile
     }
 
+  val injectCacheKeyTask =
+    injectCacheKey <<= (resourceManaged in Compile, organization, name, version, streams, cacheKey) map {
```

Comment:
Is there a reason you can't use `:=` and call `.value` on the dependencies?

I ask because I honestly can't figure out how `<<=` differs from `:=`, and I'd prefer to use `:=` if possible.